packages/@okta/vuepress-site/docs/guides/pwd-optional-widget-sign-in-email/main/java/integrationsteps.md

```diff
@@ -0,0 +1,66 @@
+The following code defines an endpoint that receives the `OTP` and `state` values from the magic link. If `state` is null or `IDX_CLIENT_CONTEXT` is not present in session state, the code assumes the callback has not come from the same browser and prompts the user accordingly.
```

Comment:
"... prompts the user accordingly" - Where is the prompt in the sample code for this step?